site/en/docs/privacy-sandbox/fedcm/index.md

```diff
@@ -216,26 +242,598 @@ They can do the same for Chrome on desktop by going to
+  we’d like to continue exploring  mechanisms to offer better privacy
+  guarantees, such as IdP blindness, directed identifiers.
+* **Relationship with WebAuthn**: As suggested by
+  [Apple](https://lists.webkit.org/pipermail/webkit-dev/2022-March/032162.html),
+  we are super excited to see the progress on
+  [passkeys](http://goo.gle/passkeys) and to work on providing a coherent and
+  cohesive experience between FedCM, Passwords, WebAuthn and WebOTP.
+* **Login Status**: As Apple suggested with the Privacy CG’s [Login Status
+  API](https://github.com/privacycg/is-logged-in), we share the intuition that
+  the user’s login status is a useful bit of information that can help browsers
+  make informed decisions, and we are excited to see what opportunities arise
+  from that.
+* **Enterprises and Education**: As is clear at the FedID CG, there are still [a
+  lot of use
+  cases](https://github.com/fedidcg/use-case-library/blob/main/decision_tree_flows/login/Federated%20Login%20OIDC%20Oauth2%20Auth%20Code%20Flow.png)
+  that are not well served by FedCM that we’d like to work on, such as  
+  front-channel logout (the ability for an IdP to send a signal to RPs to
+  logout) and support for SAML.
+* **Relationship with mDLs/VCs/etc**: continue working to understand how these
+  fit within FedCM, for example with the [Mobile Document Request
+  API](https://github.com/WICG/mobile-document-request-api).
+
+## How can identity providers develop FedCM?
+
+You need a secure context (HTTPS or localhost) in Chrome to use the FedCM.
+
+### Debug code on Chrome on Android {: #remote-debug-android}
+
+Set up and run a server locally to debug your FedCM code. You can [access this
+server in Chrome on an Android device connected using a USB cable with port
+forwarding](/docs/devtools/remote-debugging/local-server/).
+
+You can use DevTools on desktop to debug Chrome on Android by following the
+instructions at [Remote debug Android
+devices](/docs/devtools/remote-debugging/).
+
+## Use the FedCM API {: #use-api }
+
+You integrate with FedCM by creating [a well-known file](#well-known-file),
+[config file and endpoints](#idp-config-file) for [accounts
+list](#accounts-list-endpoint), [assertion issuance](#id-assertion-endpoint) and
+optionally [client metadata](#client-metadata-endpoint).
+
+From there, FedCM exposes JavaScript APIs that RPs can use to [sign
+in](#sign-into-rp) from the IdP.
+
+### Create a well-known file {: #well-known-file }
+
+To prevent [trackers from abusing the
+API](https://github.com/fedidcg/FedCM/issues/230), a well-known file must be
+served from `/.well-known/web-identity` of
+[eTLD+1](https://web.dev/same-site-same-origin/#same-site-cross-site) of the
+IdP.
+
+For example, if an IdP endpoints are served under
+`https://accounts.idp.example/sub/`, they must serve a well-known file at
+`https://idp.example/.well-known/web-identity` as well as [an IdP config
```

Comment:
This example is unclear because this depends on what's the domain. It could be accounts.idp.example